styles/colors-gallium-day.less

```diff
@@ -12,18 +12,20 @@
 @agate-highlight-s: var(--agate-highlight-s, 25%);
 @agate-highlight-l: var(--agate-highlight-l, 100%);
 @agate-background: @agate-white;
-@agate-foreground: black;
-@agate-background-active: fade(black, 15%);
+@agate-foreground: @agate-black;
+@agate-background-active: fade(@agate-black, 15%);
 @agate-foreground-active: @agate-white;
 @agate-background-focus: @agate-accent;
 @agate-foreground-focus: @agate-white;
 @agate-gradient: linear-gradient(to bottom, ~"hsla(var(--agate-accent-h, 0), var(--agate-accent-s, 0), var(--agate-accent-l, 0), 0.5)", @agate-accent);
-@agate-black:       #131313;
+@agate-black:       #000000;
 @agate-darker-gray: #373737;
 @agate-dark-gray:   #5e5e5e;
 @agate-gray:        #8a8a8a;
 @agate-medium-gray: #c9c9c9;
 @agate-light-gray:  #ededed;
+@agate-seal-brown:  #131313;
+@agate-night-rider: #333333;
```

Comment:
According the @Djspaceg's comment - https://github.com/enactjs/agate/pull/273#discussion_r454719858, how about @agate-night-rider-gray?